build.zig.zon

```diff
@@ -0,0 +1,7 @@
+.{
+    .name = "swayipc",
+    .version = "0.0.0",
```

Comment:
Idk, version 0.0.0 just seems wrong, maybe 0.2.0?

Reply:
Heh, works for me!